dart/lib/src/sentry_client.dart

```diff
@@ -199,7 +204,7 @@ class SentryClient {
   }
 
   /// Reports the [envelope] to Sentry.io.
-  Future<SentryId?> captureEnvelope(SentryEnvelope envelope) {
+  Future<SentryId> captureEnvelope(SentryEnvelope envelope) {
```

Comment:
not every envelope has an id, so it should be nullable

Reply:
`Transport.send` can't return a `SentryId?`, so I assume that's wrong too?

Reply:
probably, `captureEnvelope` should always return `SentryId?`, but eg `captureMessage`, `captureEvent`, etcc should return `SentryId` because it defaults to `SentryId.empty()` if not set.
but `captureEnvelope` can capture eg an attachment that does not require a `sentryId` that's why its nullable and don't default to `SentryId.empty()`